benchmarking/definitions/definition_lcbench.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from syne_tune.config_space import randint, lograndint, uniform, loguniform
+
+from benchmarking.blackbox_repository.conversion_scripts.scripts.lcbench.lcbench \
+    import METRIC_ELAPSED_TIME, METRIC_ACCURACY, RESOURCE_ATTR, BLACKBOX_NAME
+
+
+DATASET_NAMES = [
+    "APSFailure",
+    "Amazon_employee_access",
+    "Australian",
+    "Fashion-MNIST",
+    "KDDCup09_appetency",
+    "MiniBooNE",
+    "adult",
+    "airlines",
+    "albert",
+    "bank-marketing",
+    "blood-transfusion-service-center",
+    "car",
+    "christine",
+    "cnae-9",
+    "connect-4",
+    "covertype",
+    "credit-g",
+    "dionis",
+    "fabert",
+    "helena",
+    "higgs",
+    "jannis",
+    "jasmine",
+    "jungle_chess_2pcs_raw_endgame_complete",
+    "kc1",
+    "kr-vs-kp",
+    "mfeat-factors",
+    "nomao",
+    "numerai28.6",
+    "phoneme",
+    "segment",
+    "shuttle",
+    "sylvine",
+    "vehicle",
+    "volkert",
+]
+
+
+# This configuration space requires a surrogate to be used. This is needed
+# anyway, since the blackbox is not densely evaluated on a grid
+_config_space = {
```

Comment:
Could we import the config-space defined in lcbench rather to avoid the duplication?

Reply:
OK.

Reply:
OK, done. I also added a fix